### PR TITLE
Update heading and style formatting

### PR DESCRIPTION
**Pull request by shrielenee**
Remove links from headings.

This is mostly a test :)

## Purpose of this pull request

This pull request (PR) removes links from headings and places them in the actual section.

## Affected documentation pages

- https://docs.magento.com/m2/ce/user_guide/catalog/categories.html

## Affected Magento editions

- [X] Open Source
- [x] Commerce
- [x] B2B

## Links to Magento source code or PRs

- ...
- ...

### CODE REVIEW

**Comment by hickskenh** [comment]
Reviewed.  made additional updates
- changed ital and click formatting
- moved shared catalog image under the header to match other sections
- updated "See" text
- updated editions checkboxes

**Comment by shrielenee** [comment]
There are no comments because you made changes to the PR in those areas, so it removed the applicable comments because now they are outdated.

File: src/catalog/catalog-menu.md [hunk 1]
```diff
@@ -4,44 +4,48 @@ redirect_from:
...
 ![]({{ site.baseurl }}{% link images/images-b2b/admin-menu-catalog.png %}){: .zoom}
-*Catalog Menu*
+_Catalog Menu_
 <!--{% endif %}-->
 
-## To display the Catalog menu
+## Display the Catalog menu
 
-On the Admin sidebar, tap <span class="btn"> Catalog </span>.
+On the Admin sidebar, click **Catalog**.
```
Comment (shrielenee) [inline src/catalog/catalog-menu.md]: @hickskenh Leslie told me to leave button classes. @leslietilling ??

File: src/catalog/catalog-menu.md [hunk 2]
```diff
@@ -4,44 +4,48 @@ redirect_from:
...
 
 ![]({{ site.baseurl }}{% link images/images/products-grid.png %}){: .zoom}
-[*Catalog products grid* ]({{ site.baseurl }}{% link catalog/products.md %})
+[_Catalog products grid_]({{ site.baseurl }}{% link catalog/products.md %})
 
-## [Products]({{ site.baseurl }}{% link catalog/products.md %})
+## Products
```
Comment (shrielenee) [inline src/catalog/catalog-menu.md]: I'm also noticing that headings are worded in diff ways. How do we want to address consistency here? Or does it matter?

Screen Shot 2019-09-16 at 7 38 20 AM

**Comment by dshevtsov** [comment]
running tests

**Comment by dshevtsov** [comment]
Please ignore my messages in this PR, they are used to test CICD pipelines.

**Comment by dshevtsov** [comment]
running tests

**Comment by devops-devdocs** [comment]
An admin must run tests on this PR before it can be merged.

**Comment by dshevtsov** [comment]
running tests

**Comment by dshevtsov** [comment]
running tests

File: src/catalog/catalog-menu.md (hunk 1 above)
Comment (leslietilling) [reply src/catalog/catalog-menu.md]: Yes, we are leaving buttons as the spans.

File: src/catalog/catalog-menu.md (hunk 2 above)
Comment (leslietilling) [reply src/catalog/catalog-menu.md]: Lots of inconsistencies we want to look at and address going forward.

**Comment by magento-cicd2** [comment]
[CLA assistant check](https://cla.dev.magento.com/magento/merchdocs?pullRequest=18) <br/>Thank you for your submission, we really appreciate it. Like many open source projects, we ask that you all sign our [Contributor License Agreement](https://cla.dev.magento.com/magento/merchdocs?pullRequest=18) before we can accept your contribution.<br/>**0** out of **2** committers have signed the CLA.<br/><br/>:x: shrielenee<br/>:x: hickskenh

**Pull request merged**